iothub/service/src/Query/QueryResponse.cs

```diff
@@ -3,194 +3,79 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.Threading;
-using System.Threading.Tasks;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using Azure;
+using Azure.Core;
 
 namespace Microsoft.Azure.Devices
 {
-    /// <summary>
-    /// An iterable set of queried items.
-    /// </summary>
-    /// <typeparam name="T">
-    /// The type of the queried items. For instance, when using a query such as "SELECT * FROM devices",
-    /// this type should be type <see cref="ClientTwin"/>. When using a query such as "SELECT * FROM devices.jobs",
-    /// this type should be type <see cref="ScheduledJob"/>.
-    /// </typeparam>
-    public class QueryResponse<T>
```

Comment:
This class used to be the "AsyncPageable" implementation we had locally, but now it is the "Response<T>" implementation since the Azure.Core AsyncPageable pages all contain the Response<T> type embedded in them